edu.wpi.first.wpilib.plugins.cpp/src/main/resources/cpp-zip/ant/build.xml

```diff
@@ -122,7 +122,7 @@
 		<prop name="Items" value="system"/>
 	</post>
 	<loadfile srcFile="sysProps.xml" encoding="UTF-16LE" property="roboRIOSysValues"/>
-	<propertyregex property="roboRIOImage" input="${roboRIOSysValues}" regexp="FRC_roboRIO_2016_v([0-9]+)" select="\1" defaultValue="ImageRegExFail"/>
+	<propertyregex property="roboRIOImage" input="${roboRIOSysValues}" regexp="FRC_roboRIO_[0-9]+_v([0-9]+)" select="\1" defaultValue="ImageRegExFail"/>
```

Comment:
Need to make the same change to https://github.com/wpilibsuite/EclipsePlugins/blob/master/edu.wpi.first.wpilib.plugins.java/src/main/resources/java-zip/ant/build.xml